wordpress-mobile/WordPress-Android
# Issue/11254 material revisions

Fixes #11254

This PR moves Post Revisions to the material theme and adds a Dark Mode support.

VIsual Preview is not part of this PR, since I will have to update Aztec color scheme first.

![comparison_revisions](https://user-images.githubusercontent.com/728822/74003403-55c8da00-4927-11ea-9a4b-2da1202dfb3d.png)

To test:
- Navigate to the post with multiple revisions, and make sure they all look good in both light and dark themes.

You can change the app theme from App Settings.

Only one engineer/designer is required to review this PR, but anyone can do it.

PR submission checklist:
